src/sokoban/Game.java

```diff
@@ -6,54 +6,124 @@ public class Game extends Canvas {
     //
     // Game objects
     //
-    public class FlagLV implements Interfaces.IBlock {
+    public class Ground implements Interfaces.IBlock {
         @Override
         public void Draw(Graphics ctx, int x, int y) {
-            Color lv_red = new Color(140, 0, 0);
-            ctx.setColor(lv_red);
-            ctx.fillRect(x, y, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT / 3);
+           Color ground_brown = new Color(73, 50, 4);
+           Color nugget_yellow = new Color(118,87,25);
+           Color grass_green= new Color (31,82,15);
```

Comment:
3-space alignment please:
```suggestion
            Color ground_brown = new Color(73, 50, 4);
            Color nugget_yellow = new Color(118,87,25);
            Color grass_green= new Color(31,82,15);
```